easybuild/easyblocks/m/molpro.py

```diff
@@ -218,6 +225,8 @@ def install_step(self):
                     r"directory .* does not exist, try to create [Y]/n\n": '',
                 }
                 run_cmd_qa(cmd, qa=qa, std_qa=stdqa, log_all=True, simple=True)
+
+            molpro_path = os.path.join(self.installdir, 'bin', 'molpro')
```

Comment:
In fact, in one case `self.installdir` was used, and in the other, `self.full_prefix`, so not the same. However, I have set a variable `destroot` and used that instead, so that only one call to `os.path.join` is needed, and can be done outside the if statement.